department-of-veterans-affairs/caseflow
# Adding special issue page

Connects #6854 

### Description
This PR is the first part of adding special issues. It creates a new special issues table that is polymorphicly associated with both the appeal and legacy appeal tables. It also creates a special issues endpoint the page can use to update and get the currently selected special issues.

The PR also creates the frontend for special issues. It currently doesn't live within the attorney & judge checkout flows but will in the next PR. 

### Testing Plan
1. Insert the page into the flow by changing `SelectDispositionsView`'s `prevPage` to `/queue/appeals/${props.match.params.appealId}/special_issues` and updating the `      draft_decision` key in AttorneyActionsDropdown to be `special_issues` instead of `dispositions`.
1. Go through attorney checkout flow, check all the boxes, click next
1. Restart the flow and ensure all the boxes are still checked.
